Harvest.Web/Controllers/Api/ExpenseController.cs

```diff
@@ -105,6 +105,7 @@ public async Task<ActionResult> GetRecentExpensedProjects()
             // get projects where user has entered an expense in the last month
             var projects = await _dbContext.Projects.AsNoTracking()
                 .Where(p => p.IsActive && p.Expenses.Any(e => e.CreatedById == user.Id && e.CreatedOn > DateTime.UtcNow.AddMonths(-1)))
+                .OrderByDescending(a => a.Expenses.Max(m => m.CreatedOn))
```

Comment:
@srkirkland Does this look correct?

Reply:
That looks a little iffy. Probably best to inspect the generated sql. When debugging you can inspect the queryable's underlying query. Well, at least you can in Visual Studio. You can also call `.ToQueryString()` to get it.

Reply:
yea, I wouldn't do this, even if it works.  The main point is just to order by "something" so the result is consistent.  I'd say best to just orderBy project created date.